.golangci.yml

```diff
@@ -67,8 +68,16 @@ linters:
             - (github.com/golangci/golangci-lint/pkg/logutils.Log).Warnf
             - (github.com/golangci/golangci-lint/pkg/logutils.Log).Errorf
             - (github.com/golangci/golangci-lint/pkg/logutils.Log).Fatalf
+    importas:
+      alias:
+        # we can add more layers here (containerlab$1$2 etc.) as we tidy/get more consistent
+        - pkg: github.com/srl-labs/containerlab/(\w+)
+          alias: containerlab$1
```

Comment:
I wonder if makeing the alias `clab$1` would make it less-ish verbose?

`containerlaberrors -> claberrros`  
`containerlabruntime -> clabruntime`

Reply:
lets call it `core` as discussed in the chat